templates/postgresql.conf.j2

```diff
@@ -0,0 +1,5 @@
+{{ ansible_managed | comment }}
+
+{% for item in postgresql_server_conf %}
```

Comment:
Provided that `postgresql_server_conf` is a list, it should be `{% for key, value in dict_item.items() %}` and then `{{ key }} = {{ value }}
`